frontend/app/services/textile-service.js

```diff
@@ -28,19 +28,17 @@
 
 module.exports = function($http, PathHelper) {
   return {
-    render: render,
     renderWithWorkPackageContext: renderWithWorkPackageContext
   };
 
-  function renderWithWorkPackageContext(workPackageId, text) {
-    return render(text, PathHelper.apiV3WorkPackagePath(workPackageId));
+  function renderWithWorkPackageContext(workPackageForm) {
+    return render(workPackageForm.links.previewMarkup.props.href);
   }
 
-  function render(text, context) {
+  function render(url) {
     return $http({
-      url: PathHelper.apiV3TextilePath(context),
+      url: url,
       method: 'POST',
-      data: text,
```

Comment:
@0xF013 We talked about that in the office, but as a reminder:
You might want to keep sending the actual data, because the backend won't know ;-)